### PR TITLE
GettingStarted.md: Remove two "forced line breaks"

### PR DESCRIPTION
**Pull request by Olf0**
… and one superfluous example.  Also slightly enhance wording at some places.

### CODE REVIEW

File: GettingStarted.md [hunk 1]
```diff
@@ -108,16 +107,17 @@ and that it follows one of the formats at https://github.com/sailfishos/rpmlint/
...
   on your device.
 * To re-trigger OBS to download the latest code, click "Trigger services" at the package overview.
 * You can configure OBS to not build certain packages against certain repositories, or disable
   certain projects from being published.  It is very flexible.  This is available from the "Repositories"
   tab.
-* OBS will resolve dependencies during the build, if packages are available in either the current project or 
-  the build target.  If you want to build a package in your home project against other packages, which are only
-  available in SailfishOS:Chum (or another repository), then these can be added to the repository configuration
-  with code as the following in the project "Meta" configuration:
+* OBS will resolve dependencies when building a package, if dependency packages are available in
```
Comment (piggz) [inline GettingStarted.md:116]: , is unnecessary here

Comment (Olf0) [reply GettingStarted.md:116]: Well, yes, but …
* the comma is not introduced or altered by this commit.
* IMO the comma increases the readability of this lengthy sentence.

Do you feel / think it has to go?

File: GettingStarted.md [hunk 2]
```diff
@@ -108,16 +107,17 @@ and that it follows one of the formats at https://github.com/sailfishos/rpmlint/
...
 * You can configure OBS to not build certain packages against certain repositories, or disable
   certain projects from being published.  It is very flexible.  This is available from the "Repositories"
   tab.
-* OBS will resolve dependencies during the build, if packages are available in either the current project or 
-  the build target.  If you want to build a package in your home project against other packages, which are only
-  available in SailfishOS:Chum (or another repository), then these can be added to the repository configuration
-  with code as the following in the project "Meta" configuration:
+* OBS will resolve dependencies when building a package, if dependency packages are available in 
+  either the current project or the build target.
+  If you want to build a package in your home project against other packages, which are only available
```
[the remark this answers is not in the document]
Comment (Olf0) [reply GettingStarted.md:118]: My points WRT this comma preceding the "which" are exactly the same as for the comma preceding the "if": It is not strictly necessary, but …
* this comma is not introduced or altered by this commit.
* IMO the comma increases the readability of this lengthy sentence.

Do you feel / think it has to go?  And if so, why (beyond being unnecessary)?

In both cases the proper solution might be to split each sentence into two or three, but a fundamental rework is beyond the scope of this PR.

File: GettingStarted.md [hunk 3]
```diff
@@ -52,8 +51,8 @@ To create your first package, perform the following steps:
 Back at the home project on OBS, you will require target repositories to build against.
 These can be added using a UI, but it is easier to add them manually.
 
-1. Click "Advanced" and "Meta".<br />
-   Note, do not confuse this with the package Meta configuration, the names are the same.
+1. Click "Advanced" and "Meta".
+   Note, do not confuse this with the package Meta configuration, although the names are the same.
```
[the remark this answers is not in the document]
Comment (Olf0) [reply GettingStarted.md:55]: Well, I do not mind, but actually fail to see the significant difference that makes, in contrast to the awkward phrasing without "even though" / "although" before and now.
But as "although" tends to sound a bit ancient, I changed it to "even though".

File: GettingStarted.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Olf0) [reply GettingStarted.md:118]: Feel free to do that, this PR is open to "maintainer edits".

**Comment by rinigus** [comment]
Where are we with this PR? @piggz are all changes done and now OK for merge?

**Comment by piggz** [comment]
Done!

**Pull request merged**